latch/types/file.py

```diff
@@ -1,15 +1,20 @@
+import re
 from os import PathLike
 from typing import Optional, Type, Union
 
+import gql
 from flytekit.core.annotation import FlyteAnnotation
 from flytekit.core.context_manager import FlyteContext, FlyteContextManager
 from flytekit.core.type_engine import TypeEngine, TypeTransformer
 from flytekit.models.literals import Literal
 from flytekit.types.file.file import FlyteFile, FlyteFilePathTransformer
+from latch_sdk_gql.execute import execute
 from typing_extensions import Annotated
 
 from latch.types.utils import _is_valid_url
 
+is_absolute_node_path = re.compile(r"^latch://\d+.node$")
```

Comment:
```suggestion
is_absolute_node_path = re.compile(r"^(latch)?://\d+.node(/)?$")
```

im less sure about that last slash since idk if thats kosher but ¯\_(ツ)_/¯

Reply:
it's fine, `latch://12345.node/a/b/c` is valid too